modules/react/src/utils/position-children-under-views.js

```diff
@@ -0,0 +1,57 @@
+import {createElement} from 'react';
+import {View, log} from '@deck.gl/core';
+import {inheritsFrom} from './inherits-from';
```

Comment:
not important, but maybe could also make these `imports` consistent `import inheritsFrom from './inherits-from';`